lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
Although most button props are not needed in dropdowns (i.e. contextual styles, size, etc)

Maybe use a regular 'button` element and no need to import b-button props?